packages/lit-dev-content/site/tutorials/content/intro-to-lit/03.md

```diff
@@ -8,13 +8,13 @@ Here we've provided a name tag component with a message and an input element. In
 
 *   **Add a declarative event listener.**
 
-    Find the input element and add this expression inside the tag:
+    Find the input element and add a declarative event listener:
 
     ```html
-    @input=${this.changeName}
+    <input @input=${this.changeName} placeholder="Enter your name">
     ```
 
-    <code>@<var>eventName</var></code> is a special syntax for adding an event handler using an expression.
+    This `@input` expression adds `this.changeName` as an event listener for the `input` event. (You can use this syntax for any event listener—just replace input with any event name and `this.changeName` with any JavaScript expression that evaluates to an event handler function.)
```

Comment:
`event handler function` -> `event handler`. Because AFAIK the only way to do 

`el.addEventListener('name', 'props', {passive: true});`

in javascript is to use an event handler object:

```
const clickHandler = {

    // This fires when the handler is called
    handleEvent(e) { console.log('clicked'); }

    // You can specify event options same as addEventListener
    capture: false;
    passive: true;
}

html`<button @click=${clickHandler}>Click Me</button>`
```

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#the_event_listener_callback

Reply:
👍

Reply:
```suggestion
    This `@input` expression adds `this.changeName` as an event listener for the `input` event. (You can use this syntax for any event listener—just replace input with any event name and `this.changeName` with any JavaScript expression that evaluates to an event handler.)
```